docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3"
+
+services:
+
+  app:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    env_file:
+      - local.env # add the non-AWS entries from aws.env.example
+    environment:
+      GOOS: linux
```

Comment:
Will this break if your dev environment isn't Linux? (also line 23)

Reply:
The environment is related to the docker container, isn't it?  But, maybe this isn't needed at all? (I think it's for building the executable)

Reply:
Good point, but yes, I think it's not needed. From my (limited) reading, `GOOS` is used for cross-compilation.

Reply:
Actually, I see now. The Codeship build process does use that.